paywall/src/unlock.js/index.js

```diff
@@ -1,7 +1,20 @@
-import { makeIframe } from './iframeManager'
+import { makeIframe, addIframeToDocument } from './iframeManager'
 import setupPostOffices from './setupPostOffices'
+import '../paywall-builder/iframe.css'
 
-const dataIframe = makeIframe(window, '/static/dataIframe.html')
-const checkoutIframe = makeIframe(window, '/checkout')
+window.onload = () => {
+  const origin = '?origin=' + encodeURIComponent(window.origin)
 
-setupPostOffices(window, dataIframe, checkoutIframe)
+  const dataIframe = makeIframe(
+    window,
+    process.env.PAYWALL_URL + '/static/dataIframe.html' + origin
```

Comment:
Let's not make case specific files. This is a nightmare to manage because many OS do different things. can we just use `data-iframe.js`?